src/display.rs

```diff
@@ -310,13 +300,38 @@ where
     <S as ToOwned>::Owned: fmt::Debug,
     &'a S: AsRef<[u8]>,
 {
+    // write the part within the styling prefix and suffix
+    fn write_inner<W: AnyWrite<Wstr = S> + ?Sized>(&self, w: &mut W) -> Result<(), W::Error> {
+        match &self.oscontrol {
+            Some(OSControl::Link { url: u }) => {
+                write!(w, "\x1B]8;;")?;
+                w.write_str(u.as_ref())?;
+                write!(w, "\x1B\x5C")?;
+                w.write_str(self.string.as_ref())?;
+                write!(w, "\x1B]8;;\x1B\x5C")
+            }
+            Some(OSControl::Title) => {
+                write!(w, "\x1B]2;")?;
+                w.write_str(self.string.as_ref())?;
+                write!(w, "\x1B\x5C")
+            }
+            Some(OSControl::Icon) => {
```

Comment:
I think we should remove the Icon and Cwd parts.

Reply:
Argh, yeah I missed that.

Reply:
OK, pushed again. Unlike last time I grep'd for `icon` and `cwd` so that should do it.